test-skripsi/test-improvement.tsx

```diff
@@ -10,7 +10,7 @@ class DataManager<T> {
 	}
 
 	removeData(item: T): boolean {
-		const index = this.data.indexOf(item);
+		const index = this.data.findIndex((i) => i === item);
```

Comment:
Using `findIndex` instead of `indexOf` for comparison is a good approach when dealing with objects or more complex data structures. However, for consistency and clarity, consider adding a comment explaining why `findIndex` is preferred in this context, especially since `T` could be of any type.